eap/eap71-tx-recovery-s2i.json

```diff
@@ -158,6 +165,47 @@
         }
     ],
     "objects": [
+        {
+            "apiVersion": "v1",
+            "kind": "Role",
+            "metadata": {
+                "name": "listing-pod-role"
+            },
+            "rules": [
```

Comment:
I assume we're adding this because it's more restrictive than the `view` role?

Reply:
yes, the point was not using the view role but allow only the least necessary minimum

Reply:
I suspect we'll have the same problem of recreating the role on subsequent template invocations.  It may be better to use `view` to avoid this problem for the time being.